support-e2e/tests/smoke/non-delivery-state-selector.test.ts

```diff
@@ -0,0 +1,43 @@
+import { test } from '@playwright/test';
+import { afterEachTasks } from '../utils/afterEachTest';
+import { setTestCookies } from '../utils/cookies';
+
+afterEachTasks(test);
+
+const countryGroupsWithStates = ['us', 'au', 'ca'];
+const nonDeliveryProducts = [
+	{ product: 'SupporterPlus', ratePlan: 'Monthly' },
+	{ product: 'Contribution', ratePlan: 'Monthly', contribution: 10 },
+];
+
+/** We require state for non-deliverable products as we use different taxes within those regions upstream */
+countryGroupsWithStates.forEach((internationalisationId) => {
```

Comment:
This is good, just wonder if it's worthy of it's own test. Do we have another that's testing the other form fields validation?

Reply:
If we start adding more fields, we lose the reason for the test, which is to describe **_why_** we collect state on non-deliverable products in certain `countryGroups`.

Reply:
I guess my question is do we need the test to be isolated or is the comment describing why ("we require state for [...] taxes") enough and it could live within another test?

Reply:
I'm trying to follow the ["isolation of tests" best practise](https://playwright.dev/docs/best-practices#make-tests-as-isolated-as-possible). There can definitely be other tests, but what heuristic do you think we could group them by?

Reply:
In this case maybe validation rules? But this test is good